inference/core/workflows/core_steps/common/serializers.py

```diff
@@ -41,8 +41,10 @@ def serialise_sv_detections(detections: sv.Detections) -> dict:
         detection_dict[X_KEY] = x1 + detection_dict[WIDTH_KEY] / 2
```

Comment:
The nature of this change makes me worrying about integrity of the whole thing after the change.
If you don't have confidence, then output is not really sv.Detections, the same with class name being empty string. Those problems must be addressed at the block level

Reply:
ok, just realised that there is this `from_lmm(...)` constructor for sv.Detections, but still - objects produced by the block will be incompatible with other blocks due to optional property

Reply:
The issue is that different vision tasks have different information. I dont really see how this is different than a an sv detection for bounding boxes having a box vs. a detection for classification or detection for segmentation. We already support optional properties in supervision and multimodal models are bound to have these incomplete detections so i dont really see a way around it. I think all blocks should assume incompleteness and we may consider a separate method for validating supervision blocks for necessary info on the receiving end rather than generation end of a supervision detection.